app/views/homes/show.html.erb

```diff
@@ -58,9 +58,12 @@
         idea because it's likely to be unique.</li>
       <li>Add links on your website to your slug. The links should be to <code>/next</code> and <code>/previous</code>, like this:
         <ul>
-          <li>https://hotlinewebring.club/&lt;YOUR-SLUG&gt;/next</li>
-          <li>https://hotlinewebring.club/&lt;YOUR-SLUG&gt;/previous</li>
-          <li>Please don't literally use YOUR-SLUG. It is already taken.</li>
+          <li>https://hotlinewebring.club/YOUR-SLUG/next</li>
+          <li>https://hotlinewebring.club/YOUR-SLUG/previous</li>
+          <li>
+            Please don't literally use YOUR-SLUG. It is already taken. Replace
+            "YOUR-SLUG" with whatever your actual slug is.
```

Comment:
What do you think about:

```suggestion
            "YOUR-SLUG" with whatever you want your actual slug to be.
```

Because this will be in the future for the implementer?